app/components/FilterMenu.tsx

```diff
@@ -1,59 +1,55 @@
 import { ReactNode } from "react";
-import { Button, Flex, Heading, Show, Hide } from "@nycplanning/streetscape";
-import { ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Button,
+  Box,
+} from "@nycplanning/streetscape";
 
-export const FilterMenu = ({
-  onClose = () => {
-    return;
-  },
-  children,
-}: FilterMenuProps) => {
-  return (
-    <Flex
-      borderRadius={"base"}
-      padding={{ base: 3, lg: 4 }}
-      background={"white"}
-      direction={"column"}
-      width={{ base: "full", lg: "21.25rem" }}
-      maxW={{ base: "21.25rem", lg: "unset" }}
-      boxShadow={"0px 8px 4px 0px rgba(0, 0, 0, 0.08)"}
+export const FilterMenu = ({ children, defaultIndex }: FilterMenuProps) => (
+  <Accordion
+    allowToggle
+    allowMultiple
+    borderRadius={"base"}
+    padding={{ base: 3, lg: 4 }}
+    background={"white"}
+    direction={"column"}
+    width={{ base: "full", lg: "21.25rem" }}
+    maxW={{ base: "21.25rem", lg: "unset" }}
+    boxShadow={"0px 8px 4px 0px rgba(0, 0, 0, 0.08)"}
+    defaultIndex={defaultIndex}
+  >
+    <AccordionItem
+      borderTopWidth="0"
+      borderBottomWidth="0 !important"
```

Comment:
Let's avoid `!importants` if at all possible. What issues are introduced if you get rid of this?

Reply:
<img width="585" alt="image" src="https://github.com/user-attachments/assets/70a12632-2a35-4608-8a8d-e13f04f19238">
Without it, the `:last-of-type` supersedes it.  Let me know if you'd prefer the extra pixel or the !important.